schema_salad/makedoc.py

```diff
@@ -76,6 +83,97 @@ def table(self, header: str, body: str) -> str:
         ).format(header, body)
 
 
+def markdown_list_hook(markdown, text, state):
+    # type: (Markdown[str, Any], str, State) -> Tuple[str, State]
+    """Patches problematic Markdown lists for later HTML generation.
```

Comment:
Great, thanks! If this is code from mistune, then we need a comment explaining that, and the copyright + license

Reply:
No, it's original code by myself.

Reply:
Oh wow, great!